src/backend/cdb/cdbtm.c

```
@@ -2373,15 +2365,10 @@ dispatchDtxCommand(const char *cmd, bool withSnapshot, bool raiseError)
 		}
 	}
 
-	/* discard the errbuf text */
-	pfree(errbuf.data);
-
 	/* Now we clean up the results array. */
 	for (i = 0; i < resultCount; i++)
 		PQclear(results[i]);
 	free(results);
-
```

if numOfFailed >0 should report a error